examples/configs/classification_cifar10.yaml

```diff
@@ -85,6 +85,7 @@ checkpoint:
   monitor: valid/F1Score
   save_last: true
   mode: max
+  onnx_to_save: true
```

Comment:
Better to call `export_to_onnx` because the operation does exactly the export thing